data.md

```diff
@@ -510,8 +510,8 @@ However, `ByteMap` is just a wrapper around regular `Map`s.
 `#setRange(BM, START, VAL, WIDTH)` writes the integer `I` to memory as bytes (little-endian), starting at index `N`.
 
 ```k
-    syntax ByteMap ::= #setRange(ByteMap, Int, Int, Int) [function]
- // ---------------------------------------------------------------
+    syntax ByteMap ::= #setRange(ByteMap, Int, Int, Int) [function, functional, smtlib(setRange)]
+ // ---------------------------------------------------------------------------------------------
```

Comment:
We technically shouldn't do this because `#set` is not functional. We should actually remove `functional` from the `#set` function.

But we have an issue tracking it, and I guess it should be OK because once `#set` is fixed, we should be OK here too.

Reply:
Sounds like the right thing to do. Thanks